person_class.rb

```diff
@@ -0,0 +1,28 @@
+class Person
```

Comment:
- Hey, you've done well, however, it is excellent to add the setters and getters method here so we can be able to read and access them. 

```
attr_reader :id
  attr_accessor :name, :age
```
- In the code above, we need to be able to read the values of those instance variables so it is easy to access them from each Person's class and to other subsequent classes as we tend to build the app. Please, kindly add this before the constructor.